docs/protocol/substrate-cli.md

```diff
@@ -132,19 +132,18 @@ We will be downloading two files for your respective operating system.
           - "0.0.0.0:30433:30433"
         restart: unless-stopped
         command: [
-          "--chain", "gemini-3e",
+          "--chain", "gemini-3f",
```

Comment:
Not this line but line 121 above still has an old node image. Farmer has been updated below.